bee-network/bee-gossip/CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 0.3.0 - 2021-11-xx
+
+- New PeerRelation variant: Discovered;
```

Comment:
Please add proper sections as in the previous releases

Reply:
will do :+1:

Reply:
Done